assets/src/edit-story/components/panels/index.js

```diff
@@ -75,68 +76,69 @@ function intersect(a, b) {
 
 export function getPanels(elements) {
   if (elements.length === 0) {
-    return [];
+    return [
+      { type: PAGE, Panel: PageBackgroundPanel },
+      { type: NO_SELECTION, Panel: NoSelectionPanel },
+    ];
   }
 
   const isBackground = elements.length === 1 && elements[0].isBackground;
 
-  // Panels to always display, independent of the selected element.
-  const sharedPanels = [{ type: ACTIONS, Panel: ActionsPanel }];
```

Comment:
Don't need this anymore at all - it was just a dummy before we had "delete with backspace", but we got that with #63.